pkg/resources/reconciler.go

```diff
@@ -197,9 +197,10 @@ func (r *Reconciler) ReconcilePullSecret(ctx context.Context, destSecretNamespac
 	return integreatlyv1alpha1.PhaseCompleted, nil
 }
 
-func (r *Reconciler) ReconcileSubscription(ctx context.Context, owner ownerutil.Owner, target marketplace.Target, operandNS []string, preUpgradeBackupExecutor backup.BackupExecutor, client k8sclient.Client) (integreatlyv1alpha1.StatusPhase, error) {
+// TODO change  catalogSourceReconciler ...CatalogSourceReconciler by just CatalogSourceReconciler
```

Comment:
```suggestion
```

This comment won't be needed anymore

Reply:
:+1: I'll clean it

Reply:
Done. I added a new commit and I haven't squashed it in order to not modify the original commit which is a cherry-pick from our fork. Squashing it would modify that commit id